server/middleware/imageUpload.js

```diff
@@ -0,0 +1,73 @@
+import multer from 'multer';
+import dotenv from 'dotenv';
+import mkdirp from 'mkdirp';
+
+dotenv.config();
+
+const cloudinary = require('cloudinary').v2;
+const fs = require('fs');
+
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => {
+    const dir = 'uploads/';
+    mkdirp(dir, err => cb(err, dir));
+  },
+  filename: (req, file, cb) => {
+    cb(null, file.originalname);
+  },
+});
+
+const fileFilter = (req, file, cb) => {
+  if (file.mimetype === 'image/jpeg' || file.mimetytpe === 'image/jpg' || file.mimetype === 'image/png') {
+    cb(null, true);
+  } else {
+    cb(null, false);
+  }
+};
+
+const upload = multer({
+  storage,
+  limits: {
+    fileSize: 1024 * 1024 * 2,
+  },
+  fileFilter,
+}).single('logoUrl');
+
+let logoUrlUploaded;
+
+cloudinary.config({
+  cloud_name: process.env.cloud_name,
```

Comment:
Please specify these environment variables in the README file, under the installation section, so a user of your repo knows they're required to supply these variables.